templates/broadcaststatus.html

```diff
@@ -20,13 +20,17 @@
 
 {{ define "content" }}
   {{ with .Data }}
-    <div class="broadcast-status container mt-2">
-      <section class="my-5">
+    <div class="broadcast-status container">
+      <div id="r-banner" info="{{ $.Meta.Templates }}"></div>
+      <section>
         <div class="row">
           <div class="col-md-12">
             <div class="row justify-content-center">
-              <div class="col-md-12 col-lg-8 text-center">
-                <h4 class="h4 pb-3">{{ .JobTitle }}<br />{{ .Job.ID }}</h4>
+              <div class="col-md-12 col-lg-8">
+                <div class="page-title">
+                  <h4 class="h4">{{ .JobTitle }}</h4>
```

Comment:
Spacing is different to other headings, should probably look like `/settings`

Reply:
moved the header now to the left